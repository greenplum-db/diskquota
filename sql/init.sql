-- start_ignore
\! gpconfig -c shared_preload_libraries -v diskquota > /dev/null
-- end_ignore
\! echo $?
-- start_ignore
\! gpconfig -c diskquota.naptime -v 2 > /dev/null
-- end_ignore
\! echo $?
-- start_ignore
\! gpconfig -c max_worker_processes -v 20 > /dev/null
-- end_ignore
\! echo $?

-- start_ignore
\! gpstop -raf > /dev/null
-- end_ignore
\! echo $?

\! sleep 10
