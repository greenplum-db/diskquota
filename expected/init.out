-- start_ignore
\! gpconfig -c shared_preload_libraries -v diskquota > /dev/null
-- end_ignore
\! echo $?
0
-- start_ignore
\! gpconfig -c diskquota.naptime -v 2 > /dev/null
-- end_ignore
\! echo $?
0
-- start_ignore
\! gpconfig -c max_worker_processes -v 20 > /dev/null
-- end_ignore
\! echo $?
0
-- start_ignore
\! gpstop -raf > /dev/null
-- end_ignore
\! echo $?
0
\! sleep 10
